### PR TITLE
Added branch and hash to META-INF/MANIFEST.MF and X-Application-Id

### PR DESCRIPTION
**Pull request by shoeffner**
Added branch and hash to META-INF/MANIFEST.MF and X-Application-Id header as well as "version" field for /api/v1/ping.

As a side note: I added the *.log.zip files to the .gitignore, as I think they should be ignored...

### CODE REVIEW

**Comment by shoeffner** [comment]
@jwtodd @felixgao any comments on this?

File: pom.xml [hunk 1]
```diff
@@ -363,6 +365,9 @@ http://www.w3.org/2001/XMLSchema-instance">
                             <addDefaultImplementationEntries>true</addDefaultImplementationEntries>
                             <addDefaultSpecificationEntries>true</addDefaultSpecificationEntries>
                         </manifest>
+                        <manifestEntries>
+                            <Implementation-Build>${scmBranch}-${buildNumber}</Implementation-Build>
```
Comment (jwtodd) [inline pom.xml]: nice

**Comment by jwtodd** [comment]
:shipit:

**Comment by mans4singh** [comment]
Looks good :shipit:

**Comment by shoeffner** [comment]
Merged. I don't understand how to merge properly from the command line so that github picks it up.

**Pull request closed without merging**